samples/boards/nrf/system_off/src/main.c

```diff
@@ -55,22 +52,6 @@ int main(void)
 		return 0;
 	}
 
-	printf("Busy-wait %u s\n", BUSY_WAIT_S);
-	k_busy_wait(BUSY_WAIT_S * USEC_PER_SEC);
-
-	printf("Busy-wait %u s with UART off\n", BUSY_WAIT_S);
-	rc = pm_device_action_run(cons, PM_DEVICE_ACTION_SUSPEND);
-	k_busy_wait(BUSY_WAIT_S * USEC_PER_SEC);
-	rc = pm_device_action_run(cons, PM_DEVICE_ACTION_RESUME);
-
-	printf("Sleep %u s\n", SLEEP_S);
-	k_sleep(K_SECONDS(SLEEP_S));
-
-	printf("Sleep %u s with UART off\n", SLEEP_S);
-	rc = pm_device_action_run(cons, PM_DEVICE_ACTION_SUSPEND);
-	k_sleep(K_SECONDS(SLEEP_S));
-	rc = pm_device_action_run(cons, PM_DEVICE_ACTION_RESUME);
```

Comment:
[README.rst](https://github.com/zephyrproject-rtos/zephyr/blob/d254faa95ae12e7c8270563ff68762ad7eedb638/samples/boards/nrf/system_off/README.rst) needs to be updated accordingly.